unison-core/src/Unison/Name.hs

```diff
@@ -114,14 +114,19 @@ parent (Name txt) = case unsnoc (Text.splitOn "." txt) of
   Just ([],_) -> Nothing
   Just (init,_) -> Just $ Name (Text.intercalate "." init)
 
--- suffixes "" -> [] 
+-- suffixes "" -> []
 -- suffixes bar -> [bar]
 -- suffixes foo.bar -> [foo.bar, bar]
 -- suffixes foo.bar.baz -> [foo.bar.baz, bar.baz, baz]
+-- suffixes ".base.." -> [base.., .]
```

Comment:
Would you add this case to `parser-typechecker/tests/Unison/Core/Test/Name.hs` ?  Optionally add a note here, reminding that this is tested there.